src/isl/isl_range.rs

```diff
@@ -664,6 +681,12 @@ impl<T: PartialOrd> RangeImpl<T> {
     }
 }
 
+impl<T: Display> Display for RangeImpl<T> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), std::fmt::Error> {
+        write!(f, "[ {}, {} ]", &self.start, &self.end)
```

Comment:
suggestion—the display string would be more clear if you include `range::`
```suggestion
        write!(f, "range::[ {}, {} ]", &self.start, &self.end)
```

Reply:
I was confused on whether to use annotation or not but surely can add it.